packages/core/src/components/forms/numericStepper.tsx

```diff
@@ -0,0 +1,358 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+
+import { AbstractComponent, Classes, Keys } from "../../common";
+import * as Errors from "../../common/errors";
+import { HTMLInputProps, IIntentProps, IProps, removeNonHTMLProps } from "../../common/props";
+
+import { Button } from "../button/buttons";
+import { InputGroup } from "./inputGroup";
+
+export enum NumericStepperButtonPosition {
+    NONE,
+    LEFT,
+    RIGHT,
+    SPLIT,
+}
+
+export interface INumericStepperProps extends IIntentProps, IProps {
+
+    /**
+     * The button configuration with respect to the input field
+     * @default "right"
+     */
+    buttonPosition?: NumericStepperButtonPosition;
```

Comment:
@llorca I'd vote for `disabled` to keep buttons in place and just disable them, primarily because hiding buttons changes the width of the stepper component. Scenario: if I'm stacking Blueprint controls vertically in a sidebar or something, only to find that disabling my controls changes the width of the stepper but nothing else, my alignment would be busted and I'd be annoyed.